reference/build_helpers/cmake.rst

```diff
@@ -238,14 +238,17 @@ configure()
 
 .. code-block:: python
 
-    def configure(self, args=None, defs=None, source_folder=None, build_folder=None,
-                  cache_build_folder=None, pkg_config_paths=None)
+    def configure(self, args=None, defs=None, source_dir=None, build_dir=None,
+                  source_folder=None, build_folder=None, cache_build_folder=None,
+                  pkg_config_paths=None)
 
 Configures `CMake` project with the given parameters.
 
 Parameters:
     - **args** (Optional, Defaulted to ``None``): A list of additional arguments to be passed to the ``cmake`` command. Each argument will be escaped according to the current shell. No extra arguments will be added if ``args=None``
-    - **definitions** (Optional, Defaulted to ``None``): A dict that will be converted to a list of CMake command line variable definitions of the form ``-DKEY=VALUE``. Each value will be escaped according to the current shell and can be either ``str``, ``bool`` or of numeric type
+    - **defs** (Optional, Defaulted to ``None``): A dict that will be converted to a list of CMake command line variable definitions of the form ``-DKEY=VALUE``. Each value will be escaped according to the current shell and can be either ``str``, ``bool`` or of numeric type
+    - **source_dir** (Optional, Defaulted to ``None``): **[DEPRECATED]** Use ``source_folder`` instead.
+    - **build_dir** (Optional, Defaulted to ``None``): **[DEPRECATED]** Use ``build_folder`` instead.
```

Comment:
this only documents that `source_dir` and `build_dir` are deprecated, but doesn't say anything about their effect or difference from mentioned `source_folder`/`build_folder`.